charts/kueue/templates/components/prometheus/role.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.enablePrometheus }}
```

Comment:
Can you help to explain why this not in the rbac folder?

Reply:
ditto
https://github.com/Gekko0114/kueue/blob/main/config/components/prometheus/role.yaml